src/libs/ajax.js

```diff
@@ -705,7 +705,7 @@ export const Match = {
   findMatch: async (consentId, purposeId) => {
     const url = `${await Config.getApiUrl()}/match/${consentId}/${purposeId}`;
     const res = await fetchOk(url, Config.authOpts());
-    return res.json();
+    return res;
```

Comment:
You need to call res.json() for any request handled with fetchOK in order to parse the json response. 

Here's the Response itself
<img width="1233" alt="Screen Shot 2021-04-30 at 1 25 48 PM" src="https://user-images.githubusercontent.com/6664778/116731622-ee79c880-a9b7-11eb-9cba-221c0ab2f66d.png">

And here's the parsed json from that response 
<img width="963" alt="Screen Shot 2021-04-30 at 1 27 35 PM" src="https://user-images.githubusercontent.com/6664778/116731660-fc2f4e00-a9b7-11eb-94c1-861c1077b3dd.png">

So if you just pass in the response and save it as ```matchData``` in the components, variables reads like ```matchData.match``` or ```matchData.consentId``` will just return undefined.

That will ultimately affect the formatted DUOS Algorithm result string here
<img width="761" alt="Screen Shot 2021-04-30 at 1 34 22 PM" src="https://user-images.githubusercontent.com/6664778/116732481-dbb3c380-a9b8-11eb-95c7-7cb5cd707cae.png">

Reply:
Unexpected end of input usually means a poorly formatted json. Which election is throwing that error?

Reply:
for example, DAR-435-A-1

Reply:
Please move ☝🏽  conversation to slack for debugging.